Makefile

```diff
@@ -0,0 +1,27 @@
+dev:
+	docker-compose up
+
+test:
+	docker-compose -f docker-compose.testing.yml run --rm test
+
+setup:
+	cp -i config.js.tmpl config.js
+	cp -i config/config.json.tmpl config/config.json
+	cp -i config/recipients.sql.tmpl config/recipients.sql
+
+	docker volume create smokealarm_nodemodules
+	docker volume create smokealarm_postgres
+	docker-compose -f docker-compose.builder.yml run --rm npm_install
+
+migrate:
+	docker-compose -f docker-compose.builder.yml run --rm migrate
+
+clean:
+	-rm -f config.js
```

Comment:
I don't think we should have hyphens on the front of these.  If there's an error from an `rm -f`, we probably want to know about it.

Reply:
fixed